API_DOC.md

```diff
@@ -0,0 +1,406 @@
+# API Doc
+
+### JSON-RPC Interface
+
+JSON-RPC interface, same format with [CITA JSON-RPC](http://docs.citahub.com/en-US/cita/rpc-guide/rpc).
+
+> POST /
+
+#### params
+
+```ruby
+{
+  "jsonrpc": "2.0",
+  "id": 83,
+  "method": "blockNumber",
+  "params": []
+}
+```
+
+#### response
+
+```ruby
+{
+  "jsonrpc": "2.0",
+  "id": 83,
+  "result": "0x7169a"
+}
+```
+
+### System infos
+
+> GET /api/info/url
+
+Get the http and websocket url which ReBirth connected.
+
+#### response
+
+```ruby
+{
+  result: {
+    http_url: "http://localhost:1337",
+    ws_url: "http://localhost:4337"
+  }
+}
+```
+
+### Blocks
+
+Get blocks info list and paginate it.
+
+> GET /v2/api/blocks
+
+#### params
+
+Also Support camelCase.
+
+```ruby
+{
+  "block_number_from": "10" or "0xa", #  number or integer
+  "block_number_to": "20", # number or integer
+  "min_transaction_count": "min transaction count", # integer
+  "max_transaction_count": "max transaction count", # integer
+  "page": "1", # default 1
+  "per_page": "10", # default 10
+
+  # offset and limit has lower priority than page and perPage
+  "offset": "1", # database offset for pagination
+  "limit": "10", # database limit for pagination
+}
+```
+
+#### response
+
+```ruby
+{
```

Comment:
Response examples above (e.g. on Line 38) are nested with two spaces, but this one and below four spaces.

Reply:
All set to 2 spaces